integration_tests/sdk/apikey_test.py

```diff
@@ -0,0 +1,6 @@
+from aqueduct import apikey
+
+def test_apikey_local_server(sp_client):
+    sdk_api_key = apikey()
+    assert(sdk_api_key == sp_client._api_client.api_key)
```

Comment:
What is this asserting? What does `client._api_client.api_key` do? Is it just executing the same function as in line 4? If so, what's the goal of this test?